walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,89 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // Sort list by events.
+    ArrayList<Event> eventList = new ArrayList(events);
+    Collections.sort(eventList, (Event e1, Event e2) -> TimeRange.ORDER_BY_START.compare(e1.getWhen(), e2.getWhen()));
+    
+    Collection<TimeRange> optionalOpenTimes = getOpenTimes(eventList, request, true);
```

Comment:
nit: for static params such as "true" that aren't associated with a variable name to help you understand what the param is, it can be helpful to add an in-line function comment like this:
`getOpenTimes(eventList, request, /* optionalAttendees= */ true);`

Totally optional, but I personally find it helpful when reading code

Reply:
Didn't think about that! Thanks